File: WebContent/app/model/RtpInfo.js
Ext.define('testbank.model.RtpInfo', {
    extend: 'Ext.data.Model',
//    requires: ['Ext.data.UuidGenerator'],
//	idgen: 'uuid',
    fields: [
        {
            name: 'id',
            type: 'string'
        },
        {
            name: 'collectionid',
            type: 'string'
        },
        {
            name: 'requesttype',
            type: 'string'
        },
        {
            name: 'cardno',
            type: 'string'
        },
        {
            name: 'phoneno',
            type: 'string'
        },
        {
            name: 'time',
            type: 'string'
        }
    ]
});

File: src/com/testbank/model/RtpInfo.java
package com.testbank.model;

public class RtpInfo {
	private String id;
	private String collectionid;
	private String requesttype;
	private String cardno;
	private String phoneno;
	private String time;
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getTime() {
		return time;
	}
	public void setTime(String time) {
		this.time = time;
	}
	public String getCollectionid() {
		return collectionid;
	}
	public void setCollectionid(String collectionid) {
		this.collectionid = collectionid;
	}
	public String getRequesttype() {
		return requesttype;
	}
	public void setRequesttype(String requesttype) {
		this.requesttype = requesttype;
	}
	public String getCardno() {
		return cardno;
	}
	public void setCardno(String cardno) {
		this.cardno = cardno;
	}
	public String getPhoneno() {
		return phoneno;
	}
	public void setPhoneno(String phoneno) {
		this.phoneno = phoneno;
	}
}
